runtime/src/bank.rs

```diff
@@ -1636,7 +1636,7 @@ impl Bank {
             accounts_data_size_initial,
             accounts_data_size_delta_on_chain: AtomicI64::new(0),
             accounts_data_size_delta_off_chain: AtomicI64::new(0),
-            epoch_reward_status: fields.epoch_reward_status,
+            epoch_reward_status: EpochRewardStatus::default(),
```

Comment:
Can this be changed in a separate PR? It doesn't seem related to the rest of your changes.. (snapshot tests too)